### DISCUSSION TITLE
Feature for Tool

### DISCUSSION THREAD

**Discussion by ohhsodead** [bucanero/apollo-ps3#3] [General]
Hello!

Sorry to create an issue that is not, but I'd like your permission to have your package in my huge library of mods for my tool, called ModioX - it's a repo [here ](https://github.com/ohhsodead/ModioX) on my profile :)

All credits, complete readme and source links will be used as details. Nothing will be changed, and allows for users to install directly to their console. Also our database will be updated accordingly to other releases that are pushed out too.

**Comment by bucanero** [comment]
Hello @ohhsodead , thanks for your message

Sure!, 👍  feel free to add the Apollo tool to your package, just keep an eye on the `apollo-ps3` repo because I'll be probably doing some more releases as I keep fixing and adding stuff.

in any case, the app has an update-check feature, so if the ps3 user has a network connection they'll get notified of updates too. 😄 

good luck with your project!
cheers
